.github/workflows/ci_test-full.yml

```
@@ -1,4 +1,4 @@
-name: Test
+name: Test complete
```

nit: the workflow name should probably match with the filename
```suggestion
name: Test full
```